branch_1

```diff
@@ -0,0 +1 @@
+branch 1
```

Comment:
convert to MVVM and write tests.

NB but the b should be capitalized and this should be in a strings.xml ;p